pkg/globalnet/controllers/ipam/gatewaymonitor.go

```diff
@@ -220,7 +220,30 @@ func (i *GatewayMonitor) initializeIpamController() {
 	}
 
 	klog.V(4).Infof("On Gateway Node, initializing ipamController.")
-	ipamController, err := NewController(i.ipamSpec, &informerConfig)
+	clusterList, err := i.submarinerClientSet.SubmarinerV1().Clusters(i.ipamSpec.Namespace).List(metav1.ListOptions{})
+	if err != nil {
+		klog.Errorf("Error while retrieving the cluster info: %v", err)
```

Comment:
I think this would likely be a transient error so I think we should return an error and have the caller re-queue-retry.